tests/shipping/models/Pickup Invoice Address - Credit card.model.js

```diff
@@ -0,0 +1,100 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  fillEmail,
+  getRandomEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import { fillPickupAddress, goToPayment } from '../../../utils/shipping-actions'
+import {
+  completePurchase,
+  payWithCreditCard,
+} from '../../../utils/payment-actions'
+import {
+  goToInvoiceAddress,
+  fillInvoiceAddress,
+} from '../../../utils/invoice-actions'
+import { ACCOUNT_NAMES, SKUS } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Pickup Invoice Address - Credit card - ${account}`, () => {
+    beforeEach(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('should proceed with valid address only', () => {
+      const email = getRandomEmail()
+
+      setup({ skus: [SKUS.PICKUP_1_SLA], account })
+      fillEmail(email)
+      fillProfile()
+      fillPickupAddress(account)
+      goToInvoiceAddress(account)
+      fillInvoiceAddress(account)
+
+      cy.waitAndGet('#ship-number', 1000)
+        .last()
+        .clear()
+        .blur()
+      cy.get('#btn-go-to-payment').should('not.exist')
+      cy.get('.ship-number .error')
+        .should('exist')
+        .contains('Campo obrigatório.')
+      cy.get('#ship-number').type('12')
```

Comment:
Could you move this piece of code to a function, so we can reuse it on the `it` block bellow?

Reply:
sure!